src/content/nl/2020/accessibility.md

```diff
@@ -0,0 +1,585 @@
+
+> "Bijschriften bevatten niet alleen dialogen, maar identificeren ook wie er spreekt en bevatten niet-spraakinformatie die via geluid wordt overgebracht, inclusief betekenisvolle geluidseffecten"
+> -<span lang="en">WCAG, Understanding Success Criterion 1.2.2: Captions</span>
+
+{{ figure_markup(
+  caption="Video's met ondertiteling.",
+  content="0,79%",
+  classes="big-number",
+  sheets_gid="1002881600",
+  sql_file="video_track_usage.sql"
+) }}
+
+Van de sites die `<video>` -elementen gebruiken, biedt slechts 0,79% ondertitels, die we aannemen op basis van de aanwezigheid van het `<track>` -element (en die verschillen van open/ingebrande bijschriften). Houd er rekening mee dat sommige websites aangepaste oplossingen hebben voor het leveren van video- en audiobijschriften aan gebruikers, en we konden deze aangepaste oplossingen niet detecteren, dus het percentage sites dat ondertiteling gebruikt, kan hoger zijn, maar dit cijfer is een indicatie van hoe onderondersteunde ondertiteling is ingeschakeld in webvideo-inhoud. We kunnen ook de kwaliteit van de gedetecteerde ondertitels niet beoordelen en of ze de volledige betekenis van de video die ze beschrijven, al dan niet nauwkeurig weergeven.
+
+#### Video automatisch afspelen
+
+Het is aantoonbaar een ontwrichtende en ongewenste gebruikerservaring om video voor alle gebruikers automatisch af te spelen en te herhalen op een website. Video kan een bron van hulpbronnen zijn voor zowel apparaatbatterijen als gegevens. In sommige gevallen kan video inhoud bevatten die verontrustend is voor gebruikers, hetzij door storende beelden weer te geven, hetzij door te worden gebruikt als aanvalsvector tegen mensen die vatbaar zijn voor aanvallen.
+
+Voor gehandicapte gebruikers zijn er aanzienlijke belemmeringen die worden veroorzaakt door automatisch afspelen en herhalen van video. Voor gebruikers van schermlezers zal een video met audio waarschijnlijk de aankondigingen verstoren en tot verwarring leiden. Voor mensen met cognitieve handicaps zoals ADHD kan video erg afleidend zijn en het vermogen van de gebruiker om de interface te gebruiken en te begrijpen onderbreken. Mensen met vestibulaire aandoeningen kunnen ook gevaarlijk worden getriggerd door video.
+
+De <span lang="en">Web Content Accessibility Guidelines</span> hebben een criterium <a hreflang="en" href="https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html">2.2.2 Pauze, Stop, Verbergen</a> die vereisen dat elke bewegende, knipperende of scrollende inhoud (inclusief video) die langer dan 5 seconden wordt afgespeeld, een mechanisme heeft om deze te pauzeren, te stoppen of te verbergen.
+
+{{ figure_markup(
+  image="common-video-attributes.png",
+  alt="De meest voorkomende `<video>` -attributen.",
+  caption="De meest voorkomende `<video>` -attributen.",
```

Comment:
Looks like this is wrong in English too (so if you could correct there that would be great!). 

The `alt` is automatically set from the caption if not provided. So the only reason to provide it, is if you want it different. And we try to have `alt`s as plaintext rather than HTML so this should be:

```suggestion
  alt="De meest voorkomende video-attributen.",
  caption="De meest voorkomende `<video>` -attributen.",
```